backend/handler/metadata/igdb_handler.py

```diff
@@ -610,15 +610,15 @@ async def get_matched_roms_by_name(
             alternative_roms_ids = []
             for rom in alternative_matched_roms:
                 alternative_roms_ids.append(
-                    rom.get("game").get("id", "")
+                    pydash.get(rom, "game.id", "")
```

Comment:
I prefer to avoid the magical `pydash.get` function, as we lose typing information. In this case, Mypy won't be able to alert us if we are trying to append the wrong value type to this list.

Reply:
In general I would agree, but I'd like to fix/refactor all these in 1 go in a future PR and remove all of them at once in igdb_handler